### ISSUE TITLE
Add OOPS! Evaluator plugin in autoupdate repo

### ISSUE THREAD

**Issue by lukasged** [protegeproject/autoupdate#14]
Hi,

I've developed a plugin for ontology evaluation as the final project for my Master's Degree. Its `update.properties` file is located [here](https://raw.githubusercontent.com/lukasged/oops-plugin/master/update.properties)

I would like to add it to the autoupdate registry.

**Comment by csnyulas** [comment]
Is this plugin compatible with Protege 5.0? Should we added it also to the (end of) https://github.com/protegeproject/autoupdate/blob/master/update-info/5.0.0/plugins.repository file?

**Comment by lukasged** [comment]
I´ve specified Protege 5.1.0 for the [dependencies ](https://github.com/lukasged/oops-plugin/blob/3c78525835d7fb936b8743d2aac77d3d9eedfc9d/pom.xml#L22) in the pom.xml file so I think it´s only compatible with Protege 5.1.0 for now (I tried to run the plugin under 5.0.0 and I couldn´t get to start the plugin´s tab). I've also used some [classes](https://github.com/lukasged/oops-plugin/blob/94e28c2c60385294cd648434d384ad754ead90d4/src/main/java/oops/ui/OOPSWorkspaceTab.java#L12) in the code that are only available for the 5.1.0 version but I think I could make it compatible with the 5.0.0 version by avoiding that code. 

By the way, I still can't see my plugin in the autoupdate, even after the merged pull request. Does it only connect to the [https://github.com/protegeproject/autoupdate/blob/master/update-info/5.0.0/plugins.repository](https://github.com/protegeproject/autoupdate/blob/master/update-info/5.0.0/plugins.repository) file? In my Protege 5.1.0 log, when I click "Check for plugins..." and then check the logs for the autoupdate entries, I can only see: 
``Processing https://raw.githubusercontent.com/protegeproject/autoupdate/master/update-info/5.0.0/plugins.repository as a plugin registry``

**Comment by lukasged** [comment]
I gave it a try to make it compatible with ``5.0.0`` but it doesn't seem trivial. I managed to make it compatible to ``5.0.0``, but that change created issues when ran on the ``5.1.0`` version. The issue is related with the changes introduced in this [issue](https://github.com/protegeproject/protege/issues/506). My plugin's UI is based on the ``Entities`` tab UI (check the image at the bottom of the comment), and the mentioned issue changes its behaviour related to the selection of entities.

I think that for now I'll have to leave it compatible with ``5.0.0`` only, but will it appear in the autoupdate?

![image](https://cloud.githubusercontent.com/assets/5601295/22268064/5bd61f26-e286-11e6-86d4-512b3be2b1bc.png)

**Comment by csnyulas** [comment]
Don't worry about 5.0. It was only an idea, in case it was trivial. We encourage people to install the latest Protégé anyway, and having this plug-in available for 5.1 only, will provide an extra motivation to our users. I can't think of any reason for why someone having v5.0 installed would not be able to upgrade to v5.1 (5.1 does not introduce any additional system requirement on top of 5.0).

**Comment by lukasged** [comment]
Thanks for the comments @csnyulas , but did I did something wrong with the pull request related to my plugin being added to the autoupdate? At least in my case, it doesn't show up in the suggested plugins list from the autoupdate.

**Comment by csnyulas** [comment]
You were right @lukasged. There was indeed a problem because we added the reference to your `update.properties` file in the wrong `plugin.repository`. It turns out that the https://github.com/protegeproject/autoupdate/blob/master/update-info/5.0.0/plugins.repository file is the one read by all Protege 5.x versions, including version 5.1.0. I discussed this briefly with @matthewhorridge, and he says that this is designed to be so, and the main `plugins.repository` file is for Protege 4.x (and he just made a note for that), and could not be moved somewhere else because of backwards compatibility.

Right now out team is extremely busy with other things, but I create an issue (#16) to revisit this.

BTW, Protégé can now find your plug-in and I can easily install it. I found a minor problem with it, that I documented here: https://github.com/lukasged/oops-plugin/issues/7
